### PR TITLE
added gatsby-plugin-offline and deleted newsletter #182

### PR DESCRIPTION
**Pull request by olavea**

### CODE REVIEW

File: src/features/onboarding/Welcome.js [hunk 1]
```diff
@@ -20,11 +20,12 @@ const Welcome = () => {
   const profile = useSelector(selectProfile)
 
   const welcomeCompleted = Boolean(parseInt(profile.welcomeCompleted))
-  const newsletterCompleted = profile.newsletter !== undefined
+  const addToHomeScreen = Boolean(parseInt(profile.welcomeCompleted))
```
Comment (raae) [inline src/features/onboarding/Welcome.js:23]: To keep with the style rename to `addToHomeScreenCompleted` and then you need to read a new value from profile. I would do `profile.addToHomeScreenCompleted`. You now read `profile.welcomeCompleted` so you will never get the home screen prompt.

File: src/features/onboarding/Welcome.js [hunk 2]
```diff
@@ -38,25 +39,13 @@ const Welcome = () => {
...
-
-      case "newsletterOn":
+      case "addToHomeScreen":
         dispatch(
           updateUser({
             email: userEmail,
             profile: {
               ...profile,
-              newsletter: "1",
+              welcomeCompleted: "1",
```
Comment (raae) [inline src/features/onboarding/Welcome.js:48]: Here again  you need to do `addToHomeScreenCompleted`.

File: src/features/onboarding/Welcome.js [hunk 3]
```diff
@@ -67,7 +56,7 @@ const Welcome = () => {
     }
   }
 
-  if (welcomeCompleted && newsletterCompleted) return null
+  if (welcomeCompleted && addToHomeScreen) return null
```
Comment (raae) [inline src/features/onboarding/Welcome.js:59]: If you have renamed as I suggested above, do `addToHomeScreenCompleted`.

File: src/features/onboarding/Welcome.js [hunk 4]
```diff
@@ -82,14 +71,13 @@ const Welcome = () => {
...
         <Alert onClose={handleClose("newsletterOff")}>
-          <AlertTitle severity="info">Newsletter</AlertTitle>
+          <AlertTitle severity="info">Add POW! To Your Home Screen </AlertTitle>
           <Typography component="div">
             <p>
-              POW! is still a young app, and will probably change a lot in the
-              coming year. Would you like to keep up by signing up for the
-              newsletter?
+              POW! will feel more app-like if you
+              add POW! to your home screen
```
Comment (raae) [inline src/features/onboarding/Welcome.js:80]: Add a link to `https://www.usepow.app/support/install`

**Review by raae — CHANGES_REQUESTED** [review]
Also remove the offline plugin, we are not ready for it yet.

**Comment by olavea** [comment]
How to add a href hypertext reference? 

I found it in Nick organ's ( @skilldrick ) book Javascript For Kids page 86

**Comment by olavea** [comment]
<link rel="icon"
        href="data:image/svg+xml,
        <svg xmlns=%22http://w3.org/2000/svg%22 viewBox=%220 0 100 100%22>
        <text y=%22.9em%22 font-size=%2290%22>💩</text>
        </svg>" />

**Comment by raae** [comment]
Looks good, will have to test properly before accepting.

File: src/features/onboarding/Welcome.js [hunk 5]
```diff
@@ -36,38 +37,22 @@ const Welcome = () => {
...
-            email: userEmail,
-            profile: {
-              ...profile,
-              newsletter: "0",
-            },
-          })
-        )
-        break
+      case "addToHomeScreen": {
+        // Don't do some A2HS magic
```
Comment (raae) [inline src/features/onboarding/Welcome.js:44]: You need to do exactly the same as in the welcome case. Do not look at the old newsletter case as I tried to do something smart there that never really worked.

File: src/features/auth/slice.js [hunk 6]
```diff
@@ -174,7 +174,7 @@ export const selectIsPayingUser = createSelector(
   [selectProtectedProfile],
   (protectedProfile) => {
     const customerId = protectedProfile.stripeCustomerId
-    return Boolean(customerId)
+    return true
```
Comment (raae) [inline src/features/auth/slice.js:177]: Remove this please

File: src/features/onboarding/Welcome.js [hunk 7]
```diff
@@ -20,13 +20,14 @@ const Welcome = () => {
   const profile = useSelector(selectProfile)
 
   const welcomeCompleted = Boolean(parseInt(profile.welcomeCompleted))
-  const newsletterCompleted = profile.newsletter !== undefined
+  const addToHomeScreenCompleted = Boolean(parseInt(profile.addToHomeScreenCompleted))
+  //  const newsletterCompleted = profile.newsletter !== undefined
```
Comment (raae) [inline src/features/onboarding/Welcome.js:24]: Remove unused stuff

File: src/features/auth/slice.js (hunk 6 above)
Comment (raae) [inline src/features/auth/slice.js:177]: This should not be commited

File: src/features/onboarding/Welcome.js [hunk 8]
```diff
@@ -20,13 +20,14 @@ const Welcome = () => {
...
 
   const handleClose = (name) => (event) => {
     event.preventDefault()
-
+    console.log({name});
```
Comment (raae) [inline src/features/onboarding/Welcome.js:28]: Remove

**Review by raae — CHANGES_REQUESTED** [review]

**Comment by olavea** [comment]
How do I make `addToHomeScreenCompleted: "1"` happen?

**Comment by olavea** [comment]
Does this mean I am done with this?

**Comment by raae** [comment]
Not sure, but we'll start over.

**Pull request closed without merging**